static/scss/partials/dashboard.scss

```diff
@@ -209,9 +211,35 @@
     }
 
     input {
+        color: $color-grey-50;
+        display: flex;
+        font-weight: 500;
+        align-items: center;
+        background: transparent;
+        font-size: 1em;
+        padding: 6px $spacing-lg 6px 6px;
+        width: $content-xs;
         text-overflow: ellipsis;
-        width: inherit;
-        max-width: 100%;
+        white-space: nowrap;
+        overflow: hidden;
+        transition: all .2s ease;
+        // Reset styles from protocol
+        background-color: none;
+        border-radius: 0;
+        border: 0;
+        margin: 0;
+        list-style: inherit;
+    }
+
+    input[type="text"] {
+        max-width: $content-xs;
+        min-width: auto;
+    }
+
+    @media screen and (max-width: $screen-sm) {
+        input {
+            width: auto;
```

Comment:
Found that on smaller devices, (iPhone X/XS),  `width: $content-xs;` still overflowed. Set the `width` back to auto so that it truncates on smaller devices.

Reply:
👍